### PR TITLE
Modal design update

### PR DESCRIPTION
**Pull request by rishichawda**
### Status :
Ready
##

### Description :

Update modal design and animation

##

### Related Issues :

#287 

## Todos
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by kaushiknishchay — CHANGES_REQUESTED** [review]
- Add Padding to Modal body
- Add bit of margin to Modal Header

Modal Close Icon:
- Can be replaced by a `&times;` html symbol instead of using a SVG.
- Add padding to it
- Can be aligned with Modal header (see screenshot)
screenshot 2018-12-31 at 1 48 45 pm

**Comment by kaushiknishchay** [comment]
- Also check the styles on Mobile view.

File: lib/modal/theme.module.scss [hunk 1]
```diff
@@ -101,15 +124,16 @@
...
-    position: absolute;
-    right: 2%;
     display: flex;
-    align-items: center;
+    width: fit-content;
     height: fit-content;
+    position: absolute;
     font-size: 20px;
     color: #1f1e1eab;
+    right: 0.5em;
```
Comment (kaushiknishchay) [inline lib/modal/theme.module.scss]: can be changed to `1em`

**Review by kaushiknishchay — CHANGES_REQUESTED** [review]

File: lib/modal/theme.module.scss (hunk 1 above)
Comment (rishichawda) [reply lib/modal/theme.module.scss]: Updated in https://github.com/Codebrahma/react-lite-ui/pull/295/commits/efc1ceeb988e53662c2e67789757eb26eef01261. Also added text clipping for long header texts.

**Review by kaushiknishchay — APPROVED** [review]

**Pull request merged**